jazkarta/shop/utils.py

```diff
@@ -30,8 +38,8 @@ def get_site():
         possible_site = aq_parent(possible_site)
     return possible_site
 
-
-get_navigation_root_url = get_portal_url
+def get_navigation_root_url():
+    return getSite().absolute_url()
```

Comment:
Sure if you let me know which ones could be redone using plone.api, I will update them.